src/ragas/llms/base.py

```diff
@@ -92,6 +92,7 @@ def generate_text(
         stop: t.Optional[t.List[str]] = None,
         callbacks: t.Optional[Callbacks] = None,
     ) -> LLMResult:
+        temperature = temperature if n == 1 else 0.3
```

Comment:
make this a part of `BaseRagasLLM` as a new function. The provide documentation so that people implementing the `BaseRagasLLM` can use it